builtin/providers/aws/resource_aws_route53_health_check.go

```diff
@@ -128,6 +133,10 @@ func resourceAwsRoute53HealthCheckCreate(d *schema.ResourceData, meta interface{
 		healthConfig.ResourcePath = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("measure_latency"); ok {
```

Comment:
Schema elements with a `Default` value do not need to be checked with `GetOk`; it will always be `ok`. Here, `healthConfig.MeasureLatency = aws.Bool(d.Get("measure_latency").(bool))` is fine. 

Also, since this is a boolean, if the user specified `false` in their configuration this block would never be entered. since `GetOk` checks for non-default values. Not that you should have known these things, just mentioning it :smile: